### DISCUSSION TITLE
Please provide an upgrade tool for the chocolatey package to replace manual updates.

### DISCUSSION THREAD

**Discussion by kjxbyz** [chocolatey/home#329] [Ideas]
Please provide an upgrade tool for the chocolatey package to replace manual updates.

**Comment by Windos** [comment]
@kjxbyz could you please elaborate on what you're asking for/suggesting here?

If I'm interpreting it correctly at the moment, the "upgrade tool" for "the chocolatey package" **is** Chocolatey, i.e. `choco upgrade chocolatey`

**Reply by kjxbyz** [reply]
No, it is about the update of third-party packages. Check whether the upstream repository has released a new version, then create a PR, the maintainer merges it into the package repository, and then releases the new version to the community.

[Edit]

Refer to https://github.com/flathub-infra/flatpak-external-data-checker

**Reply by Windos** [reply]
It sounds like you're looking for [Chocolatey-AU](https://github.com/chocolatey-community/chocolatey-au).

This repo has a lot of examples of AU being used to keep many, many, packages up to date: https://github.com/chocolatey-community/chocolatey-packages/

I also did a livestream and blog post on the topic a while back: https://blog.chocolatey.org/2024/07/automating-chocolatey-package-updates/

**Reply by kjxbyz** [reply]
Thank you
